src/main/java/seedu/address/model/person/PrevDateMet.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a Person's previous met up date.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPrevDateMet(String)}
+ */
+public class PrevDateMet {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Previous Date Met should only contain numbers and hyphens, in the format of YYYY-MM-DD";
+    //public static final String VALIDATION_REGEX = "\\d{3,}";
```

Comment:
This class has commented out codes, can I check if this is kept that way for the discussion?

Reply:
Yes it is! 😄